scripts/simulate_traffic.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * script to simulate new blocks on parity in dev mode
+ * node scripts/simulate_traffic.js 0x00a329c0648769A73afAc7F9381E08FB43dBEA72 0x009f07E268ca8f4bA9D56Bd2B5dED1fd0D51357C
```

Comment:
what are the arguments? These are just some addresses? `src` has to be user address right? can we read it automatically? Maybe we can write whole script as parameterless? get addresses from `accounts`

Reply:
We cannot write it parameterless but I agree description should be better. I will improve it.